Makefile

```diff
@@ -37,7 +38,26 @@ $(BUILD_DIR)/%.ttf: $(SRC_DIR)/%.ttf
 
 clean:
 	@rm -rf $(BUILD_DIR)
+	@rm -rf $(RELEASE_DIR)
 	@rm -f $(VTT_TTF) $(VTT_MONO_TTF)
 
 update-requirements:
 	@bash tools/update-requirements.sh
+
+release: $(TTF) $(MONO_TTF)
+	rm -rf $(RELEASE_DIR)
+	mkdir $(RELEASE_DIR)
+	cp $(BUILD_DIR)/Ubuntu-R.ttf $(RELEASE_DIR)/Ubuntu-Regular.ttf
+	cp $(BUILD_DIR)/Ubuntu-RI.ttf $(RELEASE_DIR)/Ubuntu-Italic.ttf
+	cp $(BUILD_DIR)/Ubuntu-L.ttf $(RELEASE_DIR)/Ubuntu-Light.ttf
+	cp $(BUILD_DIR)/Ubuntu-LI.ttf $(RELEASE_DIR)/Ubuntu-LightItalic.ttf
+	cp $(BUILD_DIR)/Ubuntu-M.ttf $(RELEASE_DIR)/Ubuntu-Medium.ttf
+	cp $(BUILD_DIR)/Ubuntu-MI.ttf $(RELEASE_DIR)/Ubuntu-MediumItalic.ttf
+	cp $(BUILD_DIR)/Ubuntu-B.ttf $(RELEASE_DIR)/Ubuntu-Bold.ttf
+	cp $(BUILD_DIR)/Ubuntu-BI.ttf $(RELEASE_DIR)/Ubuntu-BoldItalic.ttf
+	cp $(BUILD_DIR)/Ubuntu-C.ttf $(RELEASE_DIR)/UbuntuCondensed-Regular.ttf
+	cp $(BUILD_DIR)/UbuntuMono-R.ttf $(RELEASE_DIR)/UbuntuMono-Regular.ttf
+	cp $(BUILD_DIR)/UbuntuMono-RI.ttf $(RELEASE_DIR)/UbuntMono-Italic.ttf
+	cp $(BUILD_DIR)/UbuntuMono-B.ttf $(RELEASE_DIR)/UbuntuMono-Bold.ttf
+	cp $(BUILD_DIR)/UbuntuMono-BI.ttf $(RELEASE_DIR)/UbuntuMono-BoldItalic.ttf
+	cp LICENCE.txt $(RELEASE_DIR)/LICENSE.txt
```

Comment:
The LICENCE is named `UFL.txt` in the zip returned by the Google Fonts Download:
* https://fonts.google.com/download?family=Ubuntu

Meh.